framework/components/ecl-search-forms/ecl-search-forms.twig

```diff
@@ -22,7 +48,7 @@
     } %}
   </form>
 {% else %}
-  <form class="ecl-search-form">
+  <form class="{{ _css_class }}" {{ _extra_attributes }}>
     <label class="ecl-search-form__textfield-wrapper">
       <span class="ecl-sr-only">Search this website</span>
```

Comment:
I was focusing on the site-headers, not the search-forms particularly, otherwise I would have rewritten it completely 😇